sidebars.js

```diff
@@ -131,6 +131,11 @@ module.exports = {
             'end-user/policies/shared-resource',
             'end-user/policies/apply-once',
             'end-user/policies/gc',
+            'how-to/dashboard/config/helm-repo',
+            'how-to/dashboard/config/image-registry',
+            'tutorials/access-application',
+            'tutorials/cloud-shell',
+            'tutorials/vela-top',
```

Comment:
vela top should be in `Day-2 Operations` instead of `CD Policies`.

Reply:
OK, I have fixed it.